index.js

```diff
@@ -7,10 +7,13 @@ module.exports = fp(
       root: path.join(__dirname, 'dist'),
       prefix: '/fastify-overview-ui/'
     })
-    fastify.get('/json-overview-ui', async () => fastify.overview())
+    await fastify.get('/fastify-overview-ui', (req, reply) => {
+      reply.sendFile('index.html')
```

Comment:
The static plugin was enough for me: I used a custom application by installing the `fastify-overview + ui` plugins.

Could you tell me when this is not working? How do you start the project locally?